test/utils/asserts.hpp

```diff
@@ -81,6 +81,8 @@ MATCHER_P(CResult, value, "")  // NOLINT(readability-redundant-string-init)
     }                                                                                        \
     (void)0
 
+std::ostream& operator<<(std::ostream& os, FizzyExecutionResult);
```

Comment:
Should this be moved closer to the other definition or left here as it is on the top-level namespace?

Reply:
Because the intention of these is print tests failures only, it is ok to have these defined in `asserts.hpp`. And the definition is in global namespace as the `FizzyExecutionResult`.

Reply:
I mean closer to the other prototype 5 lines below it.

Reply:
They have to be in different namespaces.

Reply:
What I meant is moving this prototype **after** the other prototype so they are easy to see. Anyhow it is a minor detail.